### PR TITLE
ARIA-165 Make node name suffix UUIDs become more readable

### PR DESCRIPTION
**Pull request by mxmrlv**

### CODE REVIEW

File: aria/modeling/service_template.py [hunk 1]
```diff
@@ -549,9 +549,13 @@ def as_raw(self):
...
 
     def instantiate(self, container):
-        context = ConsumptionContext.get_thread_local()
         from . import models
-        name = context.modeling.generate_node_id(self.name)
+        if self.nodes:
+            latest_node_index = self.nodes[-1].name.rsplit('_', 1)[1]
+            index = int(latest_node_index) + 1
+        else:
+            index = 0
```
Comment (ran-z) [inline aria/modeling/service_template.py]: i'd start from index=1 actually

File: aria/modeling/service_template.py [hunk 2]
```diff
@@ -549,9 +549,13 @@ def as_raw(self):
             ('requirement_templates', formatting.as_raw_list(self.requirement_templates))))
 
     def instantiate(self, container):
-        context = ConsumptionContext.get_thread_local()
         from . import models
-        name = context.modeling.generate_node_id(self.name)
+        if self.nodes:
+            latest_node_index = self.nodes[-1].name.rsplit('_', 1)[1]
```
Comment (ran-z) [inline aria/modeling/service_template.py]: is the nodes list guaranteed to be ordered?
if not then we need to find the max

either way, i'd change `latest` to `highest`

Comment (mxmrlv) [inline aria/modeling/service_template.py]: Pretty sure sql orders according to the indexed column

File: aria/modeling/service_template.py [hunk 3]
```diff
@@ -549,9 +549,14 @@ def as_raw(self):
             ('requirement_templates', formatting.as_raw_list(self.requirement_templates))))
 
     def instantiate(self, container):
-        context = ConsumptionContext.get_thread_local()
         from . import models
-        name = context.modeling.generate_node_id(self.name)
+        import pydevd; pydevd.settrace('localhost', suspend=False)
```
Comment (ran-z) [inline aria/modeling/service_template.py]: poo

File: aria/modeling/service_template.py [hunk 4]
```diff
@@ -549,9 +549,14 @@ def as_raw(self):
...
+        if self.nodes:
+            highest_name_suffix = max(self.nodes, key=lambda n: n.name).name.rsplit('_', 1)[1]
```
Comment (ran-z) [inline aria/modeling/service_template.py]: im not sure this would work;
note that `mynode_2` will come after `mynode_11` :|

**Pull request merged**